src/service/subscriptionService.ts

```diff
@@ -0,0 +1,35 @@
+import { Article } from "skiosa-orm/lib/model/article";
+import { Feed } from "skiosa-orm/lib/model/feed";
+import { UserInfo } from "../model/jwt";
+import { PaginationArg } from "../model/paginationArg";
+
+export interface SubscriptionService {
+
+  /**
+   * @author Amos Gross
+   * @summary change subscription status
+   * @description either create or delete a subscription to the current user
+   * @param feedId - to subscribe to 
```

Comment:
currentUserInfo param missing...

Reply:
is this intentional?

Reply:
annotation is kinda unnessicary here, since our guidelines dictate that we specify this however, I'll fix this